integritybackend/config.py

```diff
@@ -65,8 +65,57 @@ def all_orgs(self):
         return self.config.keys()
 
     def get(self, org_id):
-        """Gets configuration dictionary for an org id."""
-        return self.config.get(org_id)
+        """Gets configuration dictionary for an org."""
+        if org_id in self.config:
+            return self.config.get(org_id)
+        else:
+            raise Exception(f"No organization with ID {org_id}")
+
+    def get_collections(self, org_id):
+        """Gets collection array for an org id."""
+        org_dict = self.get(org_id)
+        if "collections" in org_dict:
+            return org_dict.get("collections")
```

Comment:
Is this returning the raw, without the `conf` key? I'd expect this to be the raw schema.

Reply:
Ok config.py re-written to use json_config which is the ORIGINAL schema loaded from the file.

`get()` still uses the re-indexed version but `get_org()` which is the root of all other `get*` definitions uses cean data.